packages/cashier/src/stores/payment-agent-transfer-store.ts

```diff
@@ -91,25 +124,32 @@ export default class PaymentAgentTransferStore {
         general_store.setLoading(true);
         this.onRemount = this.onMountPaymentAgentTransfer;
         await general_store.onMountCommon();
-        if (!this.transfer_limit.min_withdrawal) {
-            const response = await payment_agent.getPaymentAgentList();
+        if (!this.transfer_limit.min) {
```

Comment:
`transfer_limit.min_withdrawal` didn’t exist and the condition was always `true`